CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.2.0 - 2024-05-08
+
+https://github.com/open-telemetry/opentelemetry-configuration/releases/tag/v0.2.0
```

Comment:
Can we include a break-down of the included changes?

```suggestion
## [v0.2.0] - 2024-05-08

* Document time value units in kitchen-sink example. [#51](https://github.com/open-telemetry/opentelemetry-configuration/pull/51)
* Add resource `schema_url`. [#50](https://github.com/open-telemetry/opentelemetry-configuration/pull/50)
* Add Prometheus without units & type suffix options. [#61](https://github.com/open-telemetry/opentelemetry-configuration/pull/61)
* Add `without_scope_info` to Prometheus exporter. [#63](https://github.com/open-telemetry/opentelemetry-configuration/pull/63)
* Add insecure field. @codeboten in [#30](https://github.com/open-telemetry/opentelemetry-configuration/pull/30)
* Add comments for each Prometheus exporter config option. [#66](https://github.com/open-telemetry/opentelemetry-configuration/pull/66)
* Remove specification for `service.name`. [#68](https://github.com/open-telemetry/opentelemetry-configuration/pull/68)
* Add console definition for log record exporter. [#84](https://github.com/open-telemetry/opentelemetry-configuration/pull/84)
* Define `resource.detectors.attributes.included/excluded`. [#64](https://github.com/open-telemetry/opentelemetry-configuration/pull/64)
* Add with_resource_constant_labels in exporter/prometheus. [#67](https://github.com/open-telemetry/opentelemetry-configuration/pull/67)
* Add schema modeling guidance to use wildcard syntax instead of regex. [#71](https://github.com/open-telemetry/opentelemetry-configuration/pull/71)

[v0.2.0]: https://github.com/open-telemetry/opentelemetry-configuration/releases/tag/v0.2.0
```

Reply:
@MrAlias we could, i wasn't sure if having it in both places makes sense or not